library/src/scripts/components/subcommunities/SubcommunityList.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * @author Stéphane LaFlèche <stephane.l@vanillaforums.com>
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import * as React from "react";
+import classNames from "classnames";
+import { t } from "@library/application";
+import Paragraph from "@library/components/Paragraph";
+import SubcommunityTile from "@library/components/subcommunities/SubcommunityTile";
+
+interface IProps {
+    className?: string;
+    items: any[];
```

Comment:
We shouldn't be using any here. I'd rather have this component just take a `children: React.ReactNode` and the parent component does the iteration.